src/Engines/MeiliSearchEngine.php

```diff
@@ -309,4 +316,18 @@ public function __call($method, $parameters)
     {
         return $this->meilisearch->$method(...$parameters);
     }
+
+    /**
+     * Get the version of the meilisearch/meilisearch-php package intalled by Composer.
+     *
+     * @return string|null
+     */
+    protected function getInstalledMeilisearchVersion()
+    {
+        try {
+            return InstalledVersions::getVersion('meilisearch/meilisearch-php');
```

Comment:
This won't work for users still on Composer v1.0

Reply:
Any recommendations on how to support 1.x as well, other than reading raw `composer.lock`?

Reply:
None I'm afraid...

Reply:
You can to read it from https://github.com/meilisearch/meilisearch-php/blob/main/src/MeiliSearch.php#L9

Reply:
Oh wow completely missed that. Will update.